pos_search_accented_unaccented/static/src/js/db.js

```diff
@@ -0,0 +1,41 @@
+/*
+    POS Search with Accented/Unaccented characters module for Odoo
+    Copyright (C) 2019 Coop IT Easy
```

Comment:
Simplify, the (c) is not needed because you already write `copyright`.

Reply:
done